src/sentry/static/sentry/app/components/assistant/guideDrawer.jsx

```diff
@@ -2,26 +2,22 @@ import PropTypes from 'prop-types';
 import React from 'react';
 import Button from '../buttons/button';
 import {t} from '../../locale';
-import {dismiss, markUseful, nextStep} from '../../actionCreators/guides';
+import {markUseful, nextStep} from '../../actionCreators/guides';
 
 // GuideDrawer is what slides up when the user clicks on a guide cue.
 export default class GuideDrawer extends React.Component {
   static propTypes = {
     guide: PropTypes.object.isRequired,
     step: PropTypes.number.isRequired,
     onClose: PropTypes.func.isRequired,
+    handleDismiss: PropTypes.func.isRequired,
```

Comment:
Let's rename the prop to `onDismiss`. The convention is that if it's a handler passed by the parent, we use `on*`, and if it's a handler the component defines itself, it's `handle*`. The function name in helper.jsx doesn't need changing. The convention helps quickly figure out who's handling what.